docs/configure.md

```diff
@@ -377,7 +377,34 @@ See [the official documentation](https://kubernetes.io/docs/concepts/configurati
 
 ## Configure a storage class
 
-Sourcegraph relies on the default storage class of your cluster. If your cluster does not have a default storage class or if you wish to use a different storage class for Sourcegraph, then you need to update all PersistentVolumeClaims with the name of the desired storage class.
+**Note:** The storage class configuration steps should be done _before_ running `./kubectl-apply-all.sh` for the first time.
+
+By default, Sourcegraph uses the default storage class of your cluster. However, **we highly recommend that you use a storage class that uses SSDs as the underlying disk type**. The configuration details for the storage class differs depending on your hosting provider, so you should:
+
+1. Create a stub base/default.StorageClass.yaml.
+
+   ```yaml
```

Comment:
can we provide a concrete example for GCP at least, ideally AWS too?

Reply:
Actually, this might not be strictly true. According to https://kubernetes.io/docs/concepts/storage/storage-classes/#gce, not specifying a `zone` will cause requests to round-robin between all the different zones that your nodes reside in (which might not be desirable).

Reply:
I still think an example helps.

For example you could have a commented out section that shows what a real GCP example would look like.

Reply:
The Azure storage class must be `managed-premium` https://github.com/sourcegraph/sourcegraph/issues/10795